lib/matplotlib/colors.py

```diff
@@ -614,9 +614,9 @@ def __call__(self, X, alpha=None, bytes=False):
 
         rgba = np.empty(shape=xa.shape + (4,), dtype=lut.dtype)
         lut.take(xa, axis=0, mode='clip', out=rgba)
-                    #  twice as fast as lut[xa];
-                    #  using the clip or wrap mode and providing an
-                    #  output array speeds it up a little more.
+        #  twice as fast as lut[xa];
```

Comment:
This is referring to using "take" instead of indexing; I think the whole comment can be deleted now.  No one is likely to re-implement this without using "take".